contracts/hey-token.clar

```diff
@@ -6,7 +6,15 @@
 
 (define-fungible-token hey-token)
 
+;; Mint developer tokens
 (ft-mint? hey-token u10000 contract-creator)
+(ft-mint? hey-token u10000 'ST399W7Z9WS0GMSNQGJGME5JADNKN56R65VGM5KGA) ;; thomas
```

Comment:
You have my address flip-flopped with Thomas, but guess we would still both get! 😄